pkg/server/config.go

```diff
@@ -192,6 +196,29 @@ func NewConfig(opts *kcpserveroptions.CompletedOptions) (*Config, error) {
 		kubernetesinformers.WithExtraNamespaceScopedIndexers(indexers.NamespaceScoped()),
 	)
 
+	if c.Options.Cache.Enabled {
+		// TODO(p0lyn0mial): use the loopback config until we have a kubeconfig file for the cache-server
```

Comment:
@sttts wdyt?

Reply:
what's missing for that? A `--cache-kubeconfig` flag?

Reply:
yes, we need a flag and a way to create a valid kubeconfig for the cache-server

Reply:
done in https://github.com/kcp-dev/kcp/pull/2183